client/src/components/screen/SignIn.tsx

```diff
@@ -195,6 +196,9 @@ function SignIn(props: Props): ReactElement {
         }
 
         await AsyncStorage.setItem('token', token);
+        registerForPushNotificationsAsync().then((token) => {
+          if (token) { AsyncStorage.setItem('push_token', token); }
+        });
```

Comment:
`token` is used here for a different purposes. Could you kindly change `token` to `pushToken` in this paragraph?